### PR TITLE
Feature/313 update alt tags

### PR DESCRIPTION
**Pull request by Niekvandam**
## Description

A few sentences describing the overall goals of the pull request's commits.

## Type of change

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [X] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

## Checklist

-   [X] My code follows the style guidelines of this project
-   [X] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [X] I updated the changelog with an end-user readable description
-   [X] I assigned this pull request to the correct project board to update the sprint board

## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Run the website
2.  Check if images have descriptive alt tags

## Link to issue

Closes: #313

### CODE REVIEW

File: src/app/modules/project/overview/overview.component.html [hunk 1]
```diff
@@ -112,7 +112,7 @@ <h3>Filter</h3>
   <div class="col-md">
     <div class="search">
       <input type="search" placeholder="search project" [formControl]="searchControl">
-      <button><img alt="" src="assets/images/research.svg" loading="lazy"></button>
+      <button><img alt="search" src="assets/images/research.svg" loading="lazy"></button>
```
Comment (Ruby77) [inline src/app/modules/project/overview/overview.component.html:115]: I would suggest making all the alt sentences start with a capital.

Comment (Niekvandam) [reply src/app/modules/project/overview/overview.component.html]: Didn't notice, thanks! Updated now

File: CHANGELOG.md [hunk 2]
```diff
@@ -38,6 +38,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changed highlight styling to be responsive on small screens - [#195](https://github.com/DigitalExcellence/dex-frontend/issues/195)
 - Project detail URLs now show the project name instead of just a project ID - [#298](https://github.com/DigitalExcellence/dex-frontend/issues/298)
 - Updates some styling issues for more consistency in the platform - [#299](https://github.com/DigitalExcellence/dex-frontend/issues/299)
+- Updated alt tags for text-only website readers [#313](https://github.com/DigitalExcellence/dex-frontend/issues/313)
```
Comment (Ruby77) [inline CHANGELOG.md:41]: `- Updated alt tags for text-only website readers - [#313](https://github.com/DigitalExcellence/dex-frontend/issues/313)`

I think you forgot the '-'

**Review by Ruby77 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -38,6 +38,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changed highlight styling to be responsive on small screens - [#195](https://github.com/DigitalExcellence/dex-frontend/issues/195)
 - Project detail URLs now show the project name instead of just a project ID - [#298](https://github.com/DigitalExcellence/dex-frontend/issues/298)
 - Updates some styling issues for more consistency in the platform - [#299](https://github.com/DigitalExcellence/dex-frontend/issues/299)
+- Updated alt tags for text-only website readers / [#313](https://github.com/DigitalExcellence/dex-frontend/issues/313)
```
Comment (Ruby77) [inline CHANGELOG.md:41]: `- Updated alt tags for text-only website readers - [#313](https://github.com/DigitalExcellence/dex-frontend/issues/313)`

Change the '/' to '-' 😉

**Review by Ruby77 — CHANGES_REQUESTED** [review]

**Review by Ruby77 — DISMISSED** [review]
Looks good

File: src/app/components/home/home.component.html [hunk 4]
```diff
@@ -39,7 +39,7 @@ <h1>All projects in one place.</h1>
 <!--share image and button-->
 <div class="row share">
   <div class="col-lg-6 col-md-12 image-wrapper">
-    <img src="assets/images/illustrations/undraw_mobile_development_8gyo.svg" alt="" loading="lazy">
+    <img src="assets/images/illustrations/undraw_mobile_development_8gyo.svg" alt="Person working on the floor next to a phone" loading="lazy">
```
Comment (juliaslaats) [inline src/app/components/home/home.component.html:42]: Is it needed to give alt-text to decorative images?

File: src/app/modules/account/login/login.component.html [hunk 5]
```diff
@@ -23,7 +23,7 @@
 
     <!-- illustration -->
     <div class="col-lg-6 col-md-12 illustration">
-      <img alt="" src="assets/images/illustrations/undraw_Access_account_re_8spm.svg" loading="lazy">
+      <img alt="Person standing next to a login page on a phone" src="assets/images/illustrations/undraw_Access_account_re_8spm.svg" loading="lazy">
```
Comment (juliaslaats) [inline src/app/modules/account/login/login.component.html:26]: Same here, this is just decorative so might not need an alt description?

File: src/app/components/home/home.component.html (hunk 4 above)
Comment (MaxKorlaar) [reply src/app/components/home/home.component.html:42]: No, I'll elaborate on this further in the PR's discussion

**Comment by MaxKorlaar** [comment]
While I appreciate the effort, I don't find it necessary to provide image descriptions for decorative images. Screen readers read these descriptions, but they are of no use for people with visual impairments, as they add little to none real value to the rest of the page, to put it in a blunt way.

For example, they don't need to know there's an icon of a magnifying glass in the search field – they simply want to know that there's a search field and that they can submit their input from that field.

The reason we're currently using empty alt tags is simple. If you don't provide one, most screen readers will apparently read out the full URL of the image, which of course is very cumbersome for people using those screen readers.

I'd like to recommend [this short but excellent blog post](https://davidwalsh.name/accessibility-tip-empty-alt-attributes) by David Walsh for further reference.

**Review by MaxKorlaar — COMMENTED** [review]
[Please see my comment in the discussion thread](https://github.com/DigitalExcellence/dex-frontend/pull/315#issuecomment-710002887)

**Comment by Niekvandam** [comment]
PR closed, alt tags will be picked up next sprint - if possible - but does not have sufficient impact within SEO for us to focus on it right now.

**Pull request closed without merging**